.github/workflows/gamemaker_build.yml

```
@@ -24,17 +24,18 @@ jobs:
         with:
           lfs: true
-          
-      - name: fetch tags
-        run: git fetch --tags origin
+          fetch-depth: 0
+          fetch-tags: true
+
+      # Causing issues with the stable release workflow;
+      # - name: fetch tags
+      #   run: git fetch --tags origin
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)